regress/pesterTests/Authorized_keys_fileperm.Tests.ps1

```diff
@@ -83,8 +82,7 @@ Describe "Tests for authorized_keys file permission" -Tags "CI" {
 
         BeforeEach {
             $filePath = Join-Path $testDir "$tC.$tI.$fileName"            
-            $logPath = Join-Path $testDir "$tC.$tI.$logName"
-            Get-Process -Name sshd -ErrorAction SilentlyContinue | Where-Object {$_.SessionID -ne 0} | Stop-process -force -ErrorAction SilentlyContinue
+            $logPath = Join-Path $testDir "$tC.$tI.$logName"            
```

Comment:
we need to call Stop-SSHDTestDaemon to stop sshd?

Reply:
Stop-SSHDTestDaemon is called in each test cases. no need to call here